<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KortinLukija</class>
 <widget class="QMainWindow" name="KortinLukija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>392</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bodoni MT</family>
    <pointsize>9</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>KortinLukija</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelKuva1">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-5</y>
      <width>641</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSyotaKortti">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Syötä kortti</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKuva2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelKello">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPVM">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
